### ISSUE TITLE
Consider suggesting the use of chocolatey in README

### ISSUE THREAD

**Issue by jennybc** [rwinlib/base#17]
I just had a good experience installing non-Rtools prerequisites via [chocolatey](https://chocolatey.org/).

It's important to use a shell that's running as administrator. Then I did:

```
choco install miktex
choco install strawberryperl
choco install innosetup --pre
choco install patch
```

**Comment by jeroen** [comment]
Thanks. I should find out how to get Rtools in chocolately.

**Comment by jennybc** [comment]
I gather that `patch` may not be necessary if one has modern Rtools.

**Comment by jeroen** [comment]
Right. Btw it's not modern rtools, `patch.exe` has been included as of rtools 3.5 (over a year ago).
